src/Board.js

```diff
@@ -70,11 +82,12 @@ function Board({ answer }) {
           answerStr[i] !== "X" &&
           answerFreqCount[guessStr[i]] > 0
         ) {
+          shareOutput[currentRow][i] = "🟨";
```

Comment:
Isn't changing state without using setState not best practice or advisable?

Reply:
Yes. Ideally, we shouldn't be using state at all for this—it can be calculated/generated from `guess`, an existing state variable. However, doing that calculation independently at the end would involve copy-pasting the logic from `handleSubmit` and at that point we should refactor to avoid code reuse.